docs/source/installation.rst

```diff
@@ -49,6 +49,7 @@ To install MHKiT-Python using `git <https://git-scm.com/>`_::
 	cd mhkit-python
 	python setup.py develop
 
+.. python setup.py install
```

Comment:
If you run it as 'develop' you should not 'install' it. Why was this added?

Reply:
This was a private note to myself, I will remove it.